packages/expo-router/tsconfig.json

```diff
@@ -3,9 +3,19 @@
   "compilerOptions": {
     "outDir": "./build",
     "rootDir": "./src",
-    "module": "commonjs"
+    "module": "commonjs",
+    "types": [
+      "jest",
+      "jest-require",
+      "node"
```

Comment:
`typed-routes` use `node:fs` and `node:path`, so I added `node` to the types. We could also remove this can just `ts-ignore` the imports.